File: game.java
import java.util.Scanner;
import java.util.Random;

public class GuessGame {
    public static void main(String[] args) {
        System.out.println("Вгадай число від 1 до 100!");
        Random random = new Random();
        int target = random.nextInt(100) + 1;
        Scanner scanner = new Scanner(System.in);
        int attempts = 0;

        while (true) {
            System.out.print("Введіть своє число: ");
            String input = scanner.nextLine();
            attempts++;
            try {
                int guess = Integer.parseInt(input);
                if (guess < target) {
                    System.out.println("Загадане число більше!");
                } else if (guess > target) {
                    System.out.println("Загадане число менше!");
                } else {
                    System.out.println("Вітаю! Ви вгадали число " + target + " за " + attempts + " спроб.");
                    break;
                }
            } catch (NumberFormatException e) {
                System.out.println("Це не число! Спробуйте ще раз.");
            }
        }
        scanner.close();
    }
}

File: game.py
import random

def guess_game():
    print("Вгадай число від 1 до 100!")
    target = random.randint(1, 100)
    attempts = 0

    while True:
        user_input = input("Введи своє число: ")
        try:
            guess = int(user_input)
            attempts += 1
        except ValueError:
            print("Це не число! Спробуй ще раз.")
            continue

        if guess < target:
            print("Загадане число більше!")
        elif guess > target:
            print("Загадане число менше!")
        else:
            print(f"Вітаю! Ви вгадали число {target} за {attempts} спроб.")
            break

guess_game()